vllm/entrypoints/openai/serving_chat.py

```diff
@@ -147,13 +155,37 @@ async def chat_completion_stream_generator(
 
                 delta_text = output.text[len(previous_texts[i]):]
                 previous_texts[i] = output.text
```

Comment:
Don't we need to move this line down as well?

ref: https://github.com/vllm-project/vllm/blob/main/vllm/entrypoints/openai/serving_completion.py#L84

Reply:
Ah yes good call. I just added that in [this commit](https://github.com/vllm-project/vllm/pull/2918/files/c3e2a7e7fee23e71f64a2a382b4c024f0b1a4da7..1081fe190948df866b898b02fa274cf154a2c13d)